src/routes/auth/register/+page.svelte

```diff
@@ -4,10 +4,13 @@
 	import i18n from '$lib/i18n';
 	import { addToast } from '$lib/toaster.svelte.js';
 	import { goto } from '$app/navigation';
+	import { page } from '$app/stores';
 
 	const { data } = $props();
 	const { supabase } = data;
 
+	const redirect = $derived($page.url.searchParams.get('redirect') || '/');
```

Comment:
```suggestion
	const redirect = $derived($page.url.searchParams.get('redirect') || '/auth');
```
On veut pas rediriger sur le auth après un register?

Reply:
pareil